pipelines/tekton/aiedge-e2e/aiedge-e2e.tensorflow-housing.pipelinerun.yaml

```diff
@@ -9,11 +9,15 @@ spec:
   - name: model-name
     value: tensorflow-housing
   - name: model-version
-    value: "1"
+    value: "2"
   - name: s3-bucket-name
     value: rhoai-edge-models
+  - name: model-registry-hostname
+    value: "modelregistry-sample-http-default.apps.srbiswas-cluster2.dev.datahub.redhat.com"
```

Comment:
If you are doing another pipeline run make sure to just change the host name here

Reply:
Yeah I was planning to remove it, but since we don't have any shared cluster, should i just leave it as `<insert model-registry-hostname` in the example pipelinerun?

Reply:
Yeah just some example value that shows what it should be without actually working.

`modelregistry-sample-http-default.apps.username-cluster.dev.datahub.redhat.com` or something